### PR TITLE
Improve German translation

### PR DESCRIPTION
**Pull request by RedstoneDubstep**
Hi, I'm Redstone_Dubstep.
As mentioned before, I've done some tweaks on the German translation file.
This is my first pull request, so if there are any problems, please contact me.

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/de_de.json [hunk 1]
```diff
@@ -558,41 +558,41 @@
...
-    "messages.securitycraft:reinforcedSlab.cannotDoubleSlab": "Du musst diesen Block besitzen um eine Doppelstufe daraus zu machen.",
+    "messages.securitycraft:reinforcedSlab.cannotDoubleSlab": "Du musst diesen Block besitzen, um eine Doppelstufe daraus zu machen.",
     "messages.securitycraft:cameraMonitor.cannotView": "Du kannst dir keine Kamera anschauen die nicht dir gehört.",
     "messages.securitycraft:cameraMonitor.bound": "Kamera bei # mit Monitor verbunden.",
-    "messages.securitycraft:cameraMonitor.unbound": "Kamera bei # vom Monitor entbunden.",
-    "messages.securitycraft:cameraMonitor.rightclickToView": "Rechtsklicke eine Kamera sie anzuschauen.",
-    "messages.securitycraft:mrat.noSlots": "Keine freien Plätze um die Mine zu verbinden mehr verfügbar!",
+    "messages.securitycraft:cameraMonitor.unbound": "Kamera bei # vom Monitor getrennt.",
+    "messages.securitycraft:cameraMonitor.rightclickToView": "Rechtsklicke eine Kamera, um sie anzuschauen.",
+    "messages.securitycraft:mrat.noSlots": "Keine freien Plätze, um die Mine zu verbinden, mehr verfügbar!",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/de_de.json]: `Keine freien Plätze, um die Mine zu verbinden, mehr verfügbar!` -> `Für die Mine ist kein freier Platz mehr verfügbar!`

File: src/main/resources/assets/securitycraft/lang/de_de.json [hunk 2]
```diff
@@ -558,41 +558,41 @@
...
     "messages.securitycraft:mrat.cantBind": "Du kannst keine Mine verbinden, die dir nicht gehört.",
     "messages.securitycraft:mrat.bound": "Mine bei # mit der Einheit verbunden.",
-    "messages.securitycraft:mrat.unbound": "Mine bei # von der Einheit entbunden.",
-	"messages.securitycraft:srat.noSlots": "Keine freien Plätze um das Geschütz zu verbinden mehr verfügbar!",
+    "messages.securitycraft:mrat.unbound": "Mine bei # von der Einheit getrennt.",
+	"messages.securitycraft:srat.noSlots": "Keine freien Plätze, um das Geschütz zu verbinden, mehr verfügbar!",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/de_de.json]: `Keine freien Plätze, um das Geschütz zu verbinden, mehr verfügbar!` -> `Für das Geschütz ist kein freier Platz mehr verfügbar!`

File: src/main/resources/assets/securitycraft/lang/de_de.json [hunk 3]
```diff
@@ -625,82 +625,82 @@
...
-    "module.securitycraft.keypad.blacklist_module.description": "Ein Blacklist Modul in einem Nummernschloss verhindert, dass geblacklistete Spieler mit dem Schloss interagieren können.",
+    "module.securitycraft.ims.whitelist_module.description": "Ein Whitelist-Modul in einem I.M.S. erlaubt gewhitelisteten Spielern, sich innerhalb des Angriffsradius der Mine zu bewegen, ohne angegriffen zu werden.",
+    "module.securitycraft.inventory_scanner.whitelist_module.description": "Ein Whitelist-Modul in einem Inventarscanner erlaubt gewhitelistete Spieler, durch das Feld zu gehen, ohne deren Inventar überprüft zu bekommen.",
+    "module.securitycraft.inventory_scanner.smart_module.description": "Ein schlaues Modul in einem Inventarscanner erlaubt dem Scanner, auch auf Metadata und Verzauberungen Rücksicht zu nehmen.",
+    "module.securitycraft.inventory_scanner.storage_module.description": "Ein Lagermodul in einem Inventarscanner erlaubt, die aus einem Spielerinventar entferneten Items im Inventarscanner zu speichern.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/de_de.json]: `entferneten` -> `entfernten`

File: src/main/resources/assets/securitycraft/lang/de_de.json [hunk 4]
```diff
@@ -625,82 +625,82 @@
...
+    "config.securitycraft:ableToCraftKeycard2.tooltip": "Ist die Level 2 Schlüsselkarte herstellbar?",
+    "config.securitycraft:ableToCraftKeycard3": "Herstellbare Level 3 Schlüsselkarte?",
+    "config.securitycraft:ableToCraftKeycard3.tooltip": "Ist die Level 3 Schlüsselkarte herstellbarbar?",
+    "config.securitycraft:ableToCraftKeycard4": "Herstellbare Level 4 Schlüsselkarte?",
+    "config.securitycraft:ableToCraftKeycard4.tooltip": "Ist die Level 4 Schlüsselkarte herstellbar?",
+    "config.securitycraft:ableToCraftKeycard5": "Herstellbare Level 5 Schlüsselkarte?",
+    "config.securitycraft:ableToCraftKeycard5.tooltip": "Ist die Level 5 Schlüsselkarte herstellbar?",
+    "config.securitycraft:ableToCraftLUKeycard": "Herstellbare begrenzt nutzbare Schlüsselkarte?",
+    "config.securitycraft:ableToCraftLUKeycard.tooltip": "Ist die begrenzt nutzbare Schlüsselkarte herstellbar?",
+    "config.securitycraft:smallerMineExplosion": "Minenexplosionen kleiner?",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/de_de.json]: `Minenexplosionen kleiner?` -> `Kleinere Minenexplosionen?`

File: src/main/resources/assets/securitycraft/lang/de_de.json [hunk 5]
```diff
@@ -741,20 +741,20 @@
...
+	"option.securitycraft.retinal_scanner.sendMessage.description": "Sendet dieser Scanner eine Nachricht, wenn er aktiviert wird?",
 	"option.securitycraft.scanner_door.sendMessage": "Nachricht: #",
-	"option.securitycraft.scanner_door.sendMessage.description": "Sendet diese Scannertür eine Nachricht wenn sie aktiviert wird?",
+	"option.securitycraft.scanner_door.sendMessage.description": "Sendet diese Scannertür eine Nachricht, wenn sie aktiviert wird?",
 	"option.securitycraft.alarm.range": "Reichweite: #",
 	"option.securitycraft.alarm.range.description": "Wie viele Blöcke weit kann der Alarm-Sound gehört werden?",
-    "option.securitycraft.cage_trap.captureMobs": "Mobs einfangen: #",
-    "option.securitycraft.cage_trap.captureMobs.description": "Solle die Käfigfalle Mobs und Spieler einfangen, oder nur Spieler?",
+    "option.securitycraft.cage_trap.captureMobs": "Mobs einsperren: #",
+    "option.securitycraft.cage_trap.captureMobs.description": "Solle die Käfigfalle Mobs und Spieler einsperren oder nur Spieler?",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/de_de.json]: `Solle` -> `Soll`

File: src/main/resources/assets/securitycraft/lang/de_de.json [hunk 6]
```diff
@@ -363,85 +363,85 @@
...
+    "help.securitycraft.security_camera.info": "Die Überwachungskamera erlaubt dir, eine Region zu überwachen, in der du nicht bist. Platziere die Kamera und rechtsklicke sie mit einem Kameramonitor (Item), um sie mit dem Monitor zu verbinden.",
+    "help.securitycraft.reinforced_glass_pane.info": "Die verstärkte Glasscheibe ist wie eine normale Glasscheibe, jedoch kann sie nur von dem Besitzer mit einem Universellen Blockentferner entfernt werden.",
+    "help.securitycraft.reinforced_iron_bars.info": "Das verstärkte Eisengitter ist wie ein normales Eisengitter, jedoch kann es nur von dem Besitzer mit einem Universellen Blockentferner entfernt werden.",
+    "help.securitycraft.electrified_iron_fence.info": "Der elektrische Eisenzaun ist wie ein normaler Zaun, jedoch können sie nur von dem Besitzer mit einem Universellen Blockentferner entfernt werden. Es gibt gibt jedem außer dem Besitzer einen Stromschlag (tödlich!).",
```
[the remark this answers is not in the document]
Comment (RedstoneDubstep) [reply src/main/resources/assets/securitycraft/lang/de_de.json]: Yeah, I have actually overlooked this, and only changed it at the electified_iron_fence_gate
there was the same mistake with the "gibt" twice

File: src/main/resources/assets/securitycraft/lang/de_de.json [hunk 7]
```diff
@@ -453,20 +453,20 @@
...
+    "gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Jeder verstärkte Block kann wie folgt gecraftet werden, wenn nicht anders in diesem Buch erwähnt. Platziere einen normalen Block (z.B. Verstärkter Stein -> Stein) und zerbreche ihn mit einem Universellen Block Verstärker, oder rechtsklicke den Universellen Blockverstärker, lege ihn in den Slot und schließe das GUI.",
+    "gui.securitycraft:scManual.recipe.adminTool": "Das Admin-Tool ist nur durch das Kreativinventar erhältlich, falls es in der Konfigurationsdatei aktiviert ist.",
+    "gui.securitycraft:scManual.recipe.admin_tool": "Das Admin-Tool ist nur durch das Kreativinventar erhältlich, falls es in der Konfigurationsdatei aktiviert ist.",
+    "gui.securitycraft:scManual.recipe.sc_manual": "Die SecurityCraft Anleitung ist durch das Kreativinventar oder via \"/sc help\" (ohne Anführungszeichen) erhältlich. Alternativ kann man ein Buch mit einem Eisengitter zusammencraften, um die Anleitung zu bekommen.",
```
[the remark this answers is not in the document]
Comment (RedstoneDubstep) [reply src/main/resources/assets/securitycraft/lang/de_de.json]: so should I delete the line?
Or just leave it?
Or reset it completely?

[the remark this answers is not in the document]
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/de_de.json]: Just leave it, it's not used anyways, as I said :P

File: src/main/resources/assets/securitycraft/lang/de_de.json [hunk 8]
```diff
@@ -424,7 +424,7 @@
     "help.securitycraft.whitelist_module.info": "Das Whitelist-Modul verschont Spieler von einer Passworteingabe oder anderen Eigenschaften des Blocks.",
     "help.securitycraft.blacklist_module.info": "Das Blacklist-Modul ist das genaue Gegenteil zum Whitelist-Modul, z.B. blockiert es Spieler von der Eingabe eines Passwortes.",
     "help.securitycraft.harming_module.info": "Das Schadensmodul schadet Spielern, die nicht auf dem Whitelist-Modul sind, wenn diese z.B. durch einen Laser laufen.",
-    "help.securitycraft.smart_module.info": "Das schlaue Modul berücksichtigt auch Enchantments und Metadata eines Items z.B. im Inventarscanner.",
+    "help.securitycraft.smart_module.info": "Das schlaue Modul berücksichtigt auch Verzauberungs- und Metadata eines Items z.B. im Inventarscanner.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/de_de.json]: Reading over this a second time, changing it to Verzauberungs- und NBT-Daten would be best.

File: src/main/resources/assets/securitycraft/lang/de_de.json [hunk 9]
```diff
@@ -461,11 +461,11 @@
...
     "gui.securitycraft:scManual.recipe.adminTool": "Das Admin-Tool ist nur durch das Kreativinventar erhältlich, falls es in der Konfigurationsdatei aktiviert ist.",
     "gui.securitycraft:scManual.recipe.admin_tool": "Das Admin-Tool ist nur durch das Kreativinventar erhältlich, falls es in der Konfigurationsdatei aktiviert ist.",
-    "gui.securitycraft:scManual.recipe.sc_manual": "Die SecurityCraft Anleitung ist durch das Kreativinventar oder via \"/sc help\" (ohne Anführungszeichen) erhältlich. Alternativ kann man ein Buch mit einem Eisengitter zusammencraften, um die Anleitung zu bekommen.",
+    "gui.securitycraft:scManual.recipe.sc_manual": "Die SecurityCraft Anleitung ist durch das Kreativinventar oder via \"/sc help\" (ohne Anführungszeichen) erhältlich. Alternativ kann man ein Buch mit einem Eisengitter in eine Werkbank legen, um die Anleitung zu bekommen.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/de_de.json]: Good change, I like this! Couldn't think of a good way to formulate this previously.

**Pull request merged**